include/odp/api/spec/traffic_mngr.h

```diff
@@ -1455,14 +1459,14 @@ int odp_tm_node_wred_config(odp_tm_node_t tm_node,
 /** odp_tm_node_lookup() can be used to find the tm_node object created with
  * the specified name.
  *
- * @param odp_tm  Odp_tm is used to identify the TM system into which this
- *                odp_tm_node object is created.
+ * @param tm      Handle of the TM system into which this odp_tm_node object is
+ *                created.
```

Comment:
Original text has copy-paste error from create call - this is lookup.

Reply:
Fixed in V5